### PR TITLE
Improve Mesmer Clones

### PR DESCRIPTION
**Pull request by Zerthox**
This names the clone ids according using what weapon(s) they are created and prefixes the clone name with their weapon.

### CODE REVIEW

File: GW2EIEvtcParser/EvtcParser.cs [hunk 1]
```diff
@@ -870,6 +870,15 @@ private void CompleteAgents(ParserController operation)
                 }
             }
 
+            operation.UpdateProgressWithCancellationCheck("Adjusting minion names");
+            foreach (AgentItem agent in _agentData.GetAgentByType(AgentItem.AgentType.NPC))
+            {
+                if (agent.GetFinalMaster() != null)
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EvtcParser.cs:876]: ... != agent, as GetFinalMaster will return "this" if no master.

**Comment by EliphasNUIT** [comment]
Thanks!

**Comment by Zerthox** [comment]
I assume you have no clue where the ones I named "Unknown" may have come from? I assumed it would be Underwater weapons until I went and tested them.

**Comment by EliphasNUIT** [comment]
Frankly, until this PR, they were all kinda Unknown to me =D The IDs come from a big agent pool dump

**Pull request merged**